<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DelBus</class>
 <widget class="QWidget" name="DelBus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Удаление автобуса</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>471</width>
     <height>371</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QComboBox" name="comboBox"/>
    </item>
    <item>
     <widget class="QPushButton" name="delBusBtn">
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="backBtn">
      <property name="text">
       <string>Назад</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>57</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Автобус</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
